packages/client/src/router/index.js

```diff
@@ -94,11 +94,20 @@ const routes = [
       {
         path: '/tenants',
         name: 'tenants',
-        component: () => import('../views/Tenants.vue'),
+        component: () => import('../views/Organizations.vue'),
```

Comment:
It may be useful to configure this to act as a [redirect](https://router.vuejs.org/guide/essentials/redirect-and-alias#Redirect) when the feature flag is enabled, which would establish the `/organizations` route as the new canonical path while still providing support for outdated links.

Reply:
Done